galsim/vonkarman.py

```diff
@@ -82,24 +86,35 @@ class VonKarman(GSObject):
     @param gsparams          An optional GSParams argument.  See the docstring for GSParams for
                              details. [default: None]
     """
-    _req_params = { "lam" : float, "r0" : float }
-    _opt_params = { "L0" : float, "flux" : float, "scale_unit" : str, "do_delta" : bool }
-    _single_params = []
+    _req_params = { "lam" : float }
+    _opt_params = { "r0" : float, "r0_500" : float, "L0" : float, "flux" : float,
```

Comment:
If r0 and r0_500 are in single_params, then they don't need to be in opt_params.  Not actually sure if it's a problem to have them in both places, but it's at least unnecessary.